### ISSUE TITLE
can't add event listener

### ISSUE THREAD

**Issue by shorowitz** [shorowitz/GameProject#9]
@pppetrov sorry trying to paste the code. 

essentially everything works in the console, and then when I introduce jquery and dom elements, it breaks. Been trying to figure it out with others but we can't get it. will push my current code now

**Comment by cuivienor** [comment]
Hey @shorowitz can you provide some context. What exactly is not working

**Comment by shorowitz** [comment]
@pppetrov do you have time to take a look?

**Comment by cuivienor** [comment]
@shorowitz yep, but I just want you to first take a look at https://github.com/ga-students/wdi-nyc-bowie_students/wiki/How-To-Ask-A-Question and give me a bit more specific info. It will go a lot faster and its a great practice to get into. I know it might seem annoying but it is worth it, trust me :smiley:

**Comment by shorowitz** [comment]
@pppetrov 

-I cannot successfully add an event listener to call my makePlay function. I have tried adding the listener on the divs outside of the constructor, and within various different positions within the constructor where makePlay resides. Depending on the location, the handler returns an error, either noting it does not know what 'this' is or what array I am referencing. The issue is definitely a problem with scope, but I can't understand it, as I have referenced the array I am trying to access successfully in other places in the constructor. 

-https://github.com/shorowitz/GameProject/commit/0650ec04827a611b2b0474411409575a4e8ec2ab

in line 63, I am trying to just get the console to tell me what this.shownBlocksArray[x] is, and while it knows what x is, it claims this.shownBlocksArray is undefined. 

-how can I get 'x' to serve as the index of shownBlocksArray to access the corresponding Block successfully?

***sorry for all the back and forth, and apologies if the above is confusing! need to get better at writing _about_  code
